src/d3a/setup/d3a-settings.json

```diff
@@ -5,7 +5,7 @@
     "tick_length": "15s",
     "market_count": 1,
     "cloud_coverage": 0,
-    "start_date": "2021-03-25"
+    "start_date": "2021-07-28"
```

Comment:
Why is this needed?

Reply:
No, the date is only updated when you run `fab write_default_settings_file`. Does not matter really. This setup file is not used by users at all unfortunately AFAIK.